content/api/v2/canteens.md

```diff
@@ -29,6 +29,18 @@ ids
 <%= headers 200, :pagination => "canteens" %>
 <%= json :canteens %>
 
+id
+: **int** - The canteen's numeric ID, which identifys it uniquely within the API.
+
+name
+: **string** - The canteen's name.
+
+address
+: **string** - A human readable real-world locator (vulgo _street address_).
+
+coordinates
+: **list of floats** (or **null**) - The coordinates of the canteen, gievn as list of latitude and longitude in north-eastern direction (negative values imply southern or western hemisphere). Can be null if coordinates are unknown.
```

Comment:
The typo will be fixed and the pull request updated.

The described behavior of negative coordinates is how Leaflet documents it, which we use both to calculate and display the coordinates in the reference app (openmensa/openmensa). I also just tested the behavior in the reference app: there it behaves as documented here.